### ISSUE TITLE
Addon dosen't work anymore

### ISSUE THREAD

**Issue by beherit** [dtrunk90/apptabsplus#7]
After some Firefox update (I don't remember exactly the latest FF version where the addon works) the addon stop working - exclude from Tab-Switching not work, space between pinned and non-pinned tabs are missing, can hide separator.

**Comment by dtrunk90** [comment]
That's correct. I need to get some time to completely rewrite the addon as many API changes happened.
